src/utilities/createRouter.ts

```diff
@@ -135,10 +128,8 @@ export function createRouter<const T extends Routes>(routes: T, options: RouterO
     back: history.back,
     go: history.go,
     install,
-    onBeforeRouteEnter,
-    onBeforeRouteLeave,
-    onBeforeRouteUpdate,
     initialized,
+    ...routeHookMethods,
```

Comment:
This feels a little code golfy. Could easily be convinced to not rest/spread these. (This is all the `onBeforeRoute...` and `onAfterRoute...` methods.

Reply:
yeah I probably wouldn't have, but I don't feel strongly

Reply:
updated